schema_based_on_diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE patient(
+ id SERIAL PRIMARY KEY NOT NULL,
+ name VARCHAR(60),
+ date_of_birth DATE
+);
+
+CREATE TABLE medical_histories(
+ id SERIAL PRIMARY KEY NOT NULL,
+ admited_at TIMESTAMP,
+ patient_id INT REFERENCES patient(id),
+ status VARCHAR(60)
+);
+
+CREATE INDEX ON medical_histories (patient_id);
+
+CREATE TABLE treatments (
+  id SERIAL PRIMARY KEY NOT NULL,
+  type VARCHAR(60),
+  name VARCHAR(60)
+);
+
+CREATE TABLE medical_histories_treatments(
+  id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
[OPTIONAL]
- [ ] The join table doesn't need to have an identifier (id), You may consider deleting this line as this table has the only purpose to help us achieve the `many-to-many` relationship
```suggestion
 
```